.github/workflows/ci.yml

```
@@ -34,6 +34,26 @@ jobs:
       - name: Validate pnpm audit allowlist
         run: node scripts/check-audit-allowlist.js
 
+      - name: Setup Rust
+        uses: dtolnay/rust-toolchain@1.81.0
+
+      - name: Cache cargo-audit
+        uses: actions/cache@v3
```

actionlint (1.7.12)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)